### PR TITLE
Tokenization animation

### PR DESCRIPTION
**Pull request by quinnjn**
### Summary

We added animations!

## Payment entry to tokenized payment method
tokenized

## Selecting saved payment methods
methodswitch

## Selecting another way to pay
sheetopen

### Checklist

- [X] Added a changelog entry
- [X] Ran unit tests

### CODE REVIEW

File: test/unit/views/main-view.js [hunk 1]
```diff
@@ -605,22 +607,32 @@ describe('MainView', function () {
       });
 
       describe('when a payment sheet is active', function () {
+        var clock;
+
         beforeEach(function () {
-          classlist.add(this.paymentMethodsElement, 'braintree-methods--active');
+          clock = sinon.useFakeTimers();
```
Comment (EvanHahn) [inline test/unit/views/main-view.js]: Minor: could we make this `this.clock`? Saw this in a few other spots, too.

Comment (crookedneighbor) [inline test/unit/views/main-view.js]: Agreed.

File: src/scss/main.scss [hunk 2]
```diff
@@ -499,6 +594,18 @@ $curve-springy: cubic-bezier(0.43, 0.48, 0.62, 1.07);
   display: block;
 }
 
+[data-braintree-id="methods-container"] {
```
Comment (lilaconlee) [inline src/scss/main.scss]: Would slightly prefer having a `braintree-methods-container` class or something along those lines here. We've loosely been trying to keep visual styles associated with class names and the hiding and showing functionality associated with the braintree id data attributes.

We could update the `braintree-options-list` to be `braintree-options-container` to keep this structure for both views, but we can do that as a different PR to keep it in scope.

File: src/views/main-view.js [hunk 3]
```diff
@@ -80,11 +80,14 @@ MainView.prototype._initialize = function () {
 
   this.model.on('changeActivePaymentView', function (id) {
     if (id === PaymentMethodsView.ID) {
-      classlist.add(paymentMethodsViews.element, 'braintree-methods--active');
+      classlist.add(paymentMethodsViews.container, 'braintree-methods--active');
+      classlist.remove(paymentMethodsViews.container, 'braintree-methods-container--inactive');
```
Comment (lilaconlee) [inline src/views/main-view.js]: Does `braintree-methods-container--inactive` get added back in the case that the transition needs to happen again? Didn't see it but I could just be missing it.

Comment (quinnjn) [inline src/views/main-view.js]: It doesn't look like we are using this property anymore. We removed it and rechecked that it didn't effect anything.

If you have an idea on why we should keep this let us know!

File: src/scss/main.scss (hunk 2 above)
Comment (quinnjn) [inline src/scss/main.scss]: [Related converation](https://github.com/braintree/braintree-web-drop-in/pull/70#discussion_r101663983)

There was only one other property defined under here after cleaning up `braintree-methods-container--inactive`. This property was also defined in `.braintree-loader__indicator`.

So we removed this all together.

**Comment by lilaconlee** [comment]
Updated with a fix for IE and integration tests are passing

**Comment by quinnjn** [comment]
@lilaconlee I'm 👍 on changes in f22f06974726df889e5f0dd85c765c11f4b5b7ba. Hopefully this fixes the integration tests than I think it is ready for 🏅

**Pull request merged**